Makefile

```diff
@@ -115,15 +115,17 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 generate: manifests controller-gen code-generator openapi-gen helm helm-docs ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations and client-go libraries.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./api/..."
 	./hack/update-codegen.sh $(GO_CMD) $(PROJECT_DIR)/bin
-	./hack/python-sdk/gen-sdk.sh
+	# TODO: fix python-sdk generation
+	# Nevertheless, any changes to the python-sdk should first be pushed to the upstream repo.
+	# ./hack/python-sdk/gen-sdk.sh
```

Comment:
The python-sdk generation is quite tricky to fix. It requires a subset of docker/podman/java/python. And since we do not plan to make any downstream jobset changes that would affect the python-sdk or offer the python-sdk in downstream, I think it is reasonably safe to leave it out for now.

Reply:
I agree with your justification